templates/courant/mail.html.twig

```diff
@@ -1,147 +1,392 @@
+          max-width: 100% !important;
+          width: auto !important;
+        }
+      }
+
+      /* -------------------------------------
+          PRESERVE THESE STYLES IN THE HEAD
+      ------------------------------------- */
+      @media all {
+        .ExternalClass {
+          width: 100%;
+        }
+        .ExternalClass,
+        .ExternalClass p,
+        .ExternalClass span,
+        .ExternalClass font,
+        .ExternalClass td,
+        .ExternalClass div {
+          line-height: 100%;
+        }
+        .apple-link a {
+          color: inherit !important;
+          font-family: inherit !important;
+          font-size: inherit !important;
+          font-weight: inherit !important;
+          line-height: inherit !important;
+          text-decoration: none !important;
+        }
+        #MessageViewBody a {
+          color: inherit;
+          text-decoration: none;
+          font-size: inherit;
+          font-family: inherit;
+          font-weight: inherit;
+          line-height: inherit;
+        }
+        .btn-primary table td:hover {
+          background-color: #34495e !important;
+        }
+        .btn-primary a:hover {
+          background-color: #34495e !important;
+          border-color: #34495e !important;
+        }
+      }
+
+      /* extra ccs */
+
+      hr {
+          height: 5px;
+          background: #e59200;
+          border-style: solid;
+          border: none;
+        }
+
+      u {
+        text-decoration: none;
+        border-bottom: 3px solid #e59200;
+      }
+
+      .inhoudKop {
+        text-align: center;
+        color: white;
+      }
+
+      .inhoudKolom {
+        width: 33%;
+        padding: 30px 10px 20px 10px;
+      }
+
+      li {
+        list-style: none;
+      }
+
+      .link {
+        text-decoration: none;
+        text-align: center;
+        color: white;
+      }
+
+      .liLink {
+        list-style: none;
+        text-align: center;
+        color: white;
+        padding: 5px;
+      }
+
+      .box {
+        padding: 25px;
+      }
+
+      .table {
+         margin-bottom: 10px;
+         border-radius: 5px;
+      }
+
+      #id {
+        padding: 20px;
+      }
+
+    </style>
+  </head>
+  <body class="">
+      <table>
+      <tr>
+        <td>&nbsp;</td>
+        <td class="container">
+          <div class="content">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <table class="main">
+              <tr>
+                <td class="wrapper">
+
+                  <!-- LOGO en TITEL -->
+                  <table border="0" cellpadding="0" cellspacing="0" style="background: #e8e8e8">
+                    <tr style="height: 100px">
+                      <td id="logo" class="inhoud-lb" style="padding: 20px 20px 30px 40px">
+                        <img class="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/C.S.R._Delft_Logo.svg/190px-C.S.R._Delft_Logo.svg.png" style="width:100px;" />
```

Comment:
Je kan hier `https://csrdelft.nl/plaetjes/courant/beeldmerk.png` van maken, dat is hetzelfde plaatje.